test/user-test.js

```diff
@@ -5,71 +5,54 @@ import recipeData from '../src/data/recipes.js'
 
 let id;
 let username;
-let pantry;
 
 describe('User', () => {
   beforeEach(() => {
     id = 2;
     username = 'Ruth';
-    pantry = [{
```

Comment:
Couldn't we just add const user = new User(id, username) to the before each so we don't have to write it before every test. I know you hate before each lol but I think it makes a lot of sense in this case